### ISSUE TITLE
Dallas 1992 Berg River data

### ISSUE THREAD

**Issue by helendallas** [helendallas/FRC_FBIS#120]
Please can you revise attached data file into the FBIS data template
[SA Invertebrate Data for FBISv5.xlsx](https://github.com/helendallas/FRC_FBIS/files/6295140/SA.Invertebrate.Data.for.FBISv5.xlsx)
 Is on dropbox: 
https://www.dropbox.com/sh/oyedi12bigmfzza/AAAa7X8Yi4bdJYLKr7SrfUfwa?dl=0

Use the FBIS Site Code is red a,d the Taxon in red.

[Dallas 1992 Berg River.xlsx](https://github.com/helendallas/FRC_FBIS/files/6295132/Dallas.1992.Berg.River.xlsx)

Shout if you have questions

**Comment by Janette-Lumley** [comment]
@helendallas I have uploaded the spreadsheet to dropbox:
https://www.dropbox.com/s/jup0ybsmytennhl/Dallas%20H.%20and%20Day%20J.%201991.xlsx?dl=0

I have checked the data using your guidline doc. I just need to add the document upload link.

**Comment by helendallas** [comment]
@Janette-Lumley Thanks, mostly looks fine. Some notes for future:

I changed sampling over 3 days as this is what it said in the methods
Only the stones biotope was sampled with two replicates, the marginal veg was a single sample
The conductivity has not been converted, it is given a microS/cm so needs to be divided by 10 to get to mS/m. I will do this.

![image](https://user-images.githubusercontent.com/45286876/122405182-2052e880-cf80-11eb-97f6-50d85fe1e604.png)


I also revised some biotopes.

I have to wait now for Dimas to add functionality to add a reference as this has been revised.
